plugins/woocommerce/src/Admin/API/Reports/Variations/DataStore.php

```diff
@@ -491,7 +502,7 @@ public function get_data( $query_args ) {
 			if ( $query_args['extended_info'] ) {
 				$this->fill_deleted_product_name( $product_data );
 			}
-
+			$product_data = array_merge($product_data,$product_data_all_sub);
```

Comment:
Paste the data together should we want to use a single table. Alternatively, we could add a separate table and paste the data on report Download.